api_data.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import json
+import sys
+
+"""Py version check"""
+if sys.version_info[0] == 3:
+    from urllib.request import urlopen
+else: 
+	from urllib import urlopen 
+
+def output_cve_id(type=None, name=None, version=None): 
+	"""Takes as input, a package name, package version. 
+	Queries cve-search' dataset for any reported 
+	vulnerabilities of the requested package. If 
+	vulnerability exists, outputs cve-id(s).
+	"""
+	if version:
+		url = 'https://cve.circl.lu/api/search/' + name + version
+	else:
+		url = 'https://cve.circl.lu/api/search/' + name
+	
+	raw_data = urlopen(url).read()
+	data = json.loads(raw_data)
+
+	if data:
+		print ('Vulnerabilties Found:\n')
+		
+		for item in data['data']:
+			print (item['id'])
+	else:
+		print ('No vulnerabilites found')
+
+if __name__ == '__main__':
+	output_cve_id()
```

Comment:
@pombredanne I had added a line after the EOF, I think i deleted it un-consiously. Will be more alert next time